### PR TITLE
Lps 120270 - fix deactivate

### PR DESCRIPTION
**Pull request by arthurchan35**

### CODE REVIEW

File: modules/apps/portal-security/portal-security-auth-verifier/configs/com.liferay.portal.security.auth.verifier.internal.digest.authentication.configuration.DigestAuthenticationAuthVerifierConfiguration-default.config [hunk 1]
```diff
@@ -0,0 +1,3 @@
+enabled=B"true"
```
Comment (liferay-appsec) [inline modules/apps/portal-security/portal-security-auth-verifier/configs/com.liferay.portal.security.auth.verifier.internal.digest.authentication.configuration.DigestAuthenticationAuthVerifierConfiguration-default.config:1]: It looks this pull is introducing a bundled configuration file
here.


If that is the case and this pull is fixing a bug please mark this pull as 
*DO NOT BACKPORT*


Please see https://issues.liferay.com/browse/PTR-1770

File: modules/apps/portal-security/portal-security-auth-verifier/configs/com.liferay.portal.security.auth.verifier.internal.request.parameter.configuration.RequestParameterAuthVerifierConfiguration-default.config [hunk 2]
```diff
@@ -0,0 +1,3 @@
+enabled=B"true"
```
Comment (liferay-appsec) [inline modules/apps/portal-security/portal-security-auth-verifier/configs/com.liferay.portal.security.auth.verifier.internal.request.parameter.configuration.RequestParameterAuthVerifierConfiguration-default.config:1]: It looks this pull is introducing a bundled configuration file
here.


If that is the case and this pull is fixing a bug please mark this pull as 
*DO NOT BACKPORT*


Please see https://issues.liferay.com/browse/PTR-1770

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

**Comment by arthurchan35** [comment]
ci:test:relevant

**Comment by csierra** [comment]
ci:test:relevant

**Comment by csierra** [comment]
@arthurchan35, what do you think about 567c0fa and 85e6991 .  I am not sure we should have those enabled by default.

**Comment by arthurchan35** [comment]
Hey @csierra 
Okay, let's just drop the commit of adding default config files and use null check .
I have no objection.

Thanks for pointing out!

**Comment by arthurchan35** [comment]
See: https://github.com/liferay-appsec/liferay-portal/pull/239

**Pull request closed without merging**